Documentation/markdown/GitProcess.md

```diff
@@ -84,10 +84,28 @@ After successfully pushing changes to your remote repository, you should merge y
 
 5. Assign *Labels* to your *pull request*. (e.g. bug)
 
-6. Select *Create pull request*
+6. Select *Create pull request*.
 
 ## <a name="Updating"></a> Updating from main repository 
 
-(Fetch all)...
+Once you have *added a remote*, you can update your local repository with all changes made in the official repository. **It is highly recommended to update your local repository every time you start working with it. Otherwise, you will not have access to recent changes**. 
+
+1. Add all updates to your local repository: 
+
+	- In **Git Bash**, which is working in OpenInfraPlatform folder, in the command line write out *git pull tumcms development*.  
```

Comment:
```suggestion
	- In **Git Bash**, which is working in OpenInfraPlatform folder, in the command line write out:
	 > git pull tumcms development
```

Reply:
Please reformat all bash-commands to be on a new line starting with ">".

Reply:
Done